src/payments/services/certificate.service.ts

```
@@ -0,0 +1,71 @@
+  issuanceDate: string;
+  expirationDate: string;
+}
+
+@Injectable()
+export class CertificateService {
+  private readonly logger = new Logger(CertificateService.name);
+  private readonly certificateServiceUrl: string;
+
+  constructor(
+    private readonly httpService: HttpService,
+    private readonly configService: ConfigService,
```

ESLint

[error] Delete `,`

(prettier/prettier)